developers/weaviate/search/similarity.md

```diff
@@ -90,7 +86,48 @@ The output is like this:
 
 </details>
 
-## Search for a similar object
+
+## Search with image
+
+import ImgSrchPyCode from '!!raw-loader!/_includes/code/howto/search.image.py';
+import ImgSrchPyCodeV3 from '!!raw-loader!/_includes/code/howto/search.image-v3.py';
+import ImgSrchTSCode from '!!raw-loader!/_includes/code/howto/search.image.ts';
+
+You can use [`nearImage`](../api/graphql/search-operators.md#nearimage) to find objects with the nearest vector to an input image. This example uses a base64 representation of an image.
```

Comment:
The example is just below.
Do we need a link to the reference page here? (this a question, not a directive)

Reply:
Same as above.